package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "test": "mocha -R spec",
     "coverage": "istanbul cover --dir coverage node_modules/mocha/bin/_mocha",
-    "grandma": "node lib/cli.js"
+    "grandma": "node bin/_cli"
```

Comment:
Eh. This is a convenience script. All `_cli` does is resolve to the locally installed `grandma`'s `cli.js` file. So frankly, I don't really care about this one. I actually just installed it globally now and don't even use this npm script.